src/Auth/LoginContainer.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component, PropTypes } from 'react';
+import Box from 'grommet/components/Box';
+import Button from 'grommet/components/Button';
+import Footer from 'grommet/components/Footer';
+import Form from 'grommet/components/Form';
+import FormField from 'grommet/components/FormField';
+import Heading from 'grommet/components/Heading';
+import TextInput from 'grommet/components/TextInput';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { changeUsername, loginRequest } from './authModule';
+
+class LoginContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.onChangeUsername = this.onChangeUsername.bind(this);
+    this.onFormLogin = this.onFormLogin.bind(this);
+  }
+
+  onChangeUsername(event) {
```

Comment:
1.  okay,
2. `redux-form`, I survey it. If I want to use `redux-form` and `grommet`, I should customize the component.
3. `onUsernameChange`, because  another one is `onFormLogin`, it's not `onLoginForm`